add additional workload-identity role-bindings (#3940)
astronomer/docs: software/integrate-iam.md

    ```

    If Workload Identity is working, you should see a list of credentialed accounts related to your GCP service account.

> ## Granting Access to Airflow components using XCom and secrets backends {sas-that-use-backends}

Reviewer: ```suggestion
## Granting Access to Airflow components using XCom and secrets backends {#sas-that-use-backends}
```

> Astronomer Software creates a set of service-accounts for each Airflow instance it manages. The following roles can require access to Airflow Xcom backends or secrets backends to function:

Reviewer: ```suggestion
Astronomer Software creates a set of service accounts for each Airflow instance it manages. The following roles can require access to Airflow Xcom backends or secrets backends to function:
```

| Component                   | Rationale                                                                                                 |
|-----------------------------|-----------------------------------------------------------------------------------------------------------|
| `<release name>-scheduler`         | User-provided DAG code is regularly interpreted to determine what tasks are part of DAGs, and that code might incorporate references to values stored in secret storage or XCom.                  |
| `<release name>-worker`            | User-provided DAG code is interpreted at task-run time and might incorporate references to values stored in secret storage or XCom.             |

> | `<release name>-triggerer`         | User-provided DAG code is interpreted at task run time and might incorporate references to values stored in secrets storage or XCom.            |

Reviewer: "user provided code is interpreted" by Astronomer Software? By Airflow? What is doing the interpreting for each of these?

> | `<release name>-scheduler`         | User-provided DAG code is regularly interpreted by the scheduler to determine what tasks are part of DAGs, and that code might incorporate references to values stored in secret storage or XCom.                  |
> | `<release name>-worker`            | User-provided DAG code is interpreted by the worker at task run-time and might incorporate references to values stored in secret storage or XCom.             |
> | `<release name>-triggerer`         | User-provided DAG code is interpreted by the triggerer at task run-time and might incorporate references to values stored in secrets storage or XCom.            |

Reviewer: ```suggestion
| `<release name>-scheduler`         | Regularly interprets your DAG code by the scheduler to determine which tasks are part of DAGs and might incorporate references to values stored in secret storage or XCom.                  |
| `<release name>-worker`            | Regularly interprets your DAG code by the worker at task runtime and might incorporate references to values stored in secret storage or XCom.             |
| `<release name>-triggerer`         | Regularly interprets your DAG code by the triggerer at task run-time and might incorporate references to values stored in secrets storage or XCom.            |
```

is this correct?

> | `<release name>-webserver`         | Provides users a mechanism to view and set secrets and to view XCom entries.|

Reviewer: ```suggestion
| `<release name>-webserver`         | Provides a mechanism for you to view XCom entries and to view and set secrets. |
```

Again, who is the user?

> | `<release name>-cleanup`           |  Accesses task instances and other data that might include serialized references to values stored in secrets backends or XCOM backends.       |

Reviewer: The `_ -cleanup` service account accesses task instances..?
